### PR TITLE
doc/rados/config: edit auth-config-ref

### PR DESCRIPTION
**Pull request by zdover23**
Line edit doc/rados/config/auth-config-ref.rst.

https://tracker.ceph.com/issues/58485

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/configuration/auth-config-ref.rst [hunk 1]
```diff
@@ -1,142 +1,141 @@
...
 Deployment Scenarios
 ====================
 
-There are two main scenarios for deploying a Ceph cluster, which impact
-how you initially configure Cephx. Most first time Ceph users use
-``cephadm`` to create a cluster (easiest). For clusters using
-other deployment tools (e.g., Chef, Juju, Puppet, etc.), you will need
-to use the manual procedures or configure your deployment tool to
-bootstrap your monitor(s).
+How you initially configure CephX depends on your scenario. There are two main
```
Comment (anthonyeleven) [inline doc/rados/configuration/auth-config-ref.rst:25]: Suggest

There are two common strategies when deploying

File: doc/rados/configuration/auth-config-ref.rst [hunk 2]
```diff
@@ -1,142 +1,141 @@
...
+scenarios for deploying a Ceph cluster.  If you are a first-time Ceph user, you
+should probably take the easiest approach: using ``cephadm`` to deploy a
+cluster. But if your cluster uses other deployment tools (for example, Chef,
+Juju, or Puppet), you will need either to use the manual deployment procedures
```
Comment (anthonyeleven) [inline doc/rados/configuration/auth-config-ref.rst:29]: Add Ansible to the list, I have to believe that more people use it to deploy Ceph than use Chef.

File: doc/rados/configuration/auth-config-ref.rst [hunk 3]
```diff
@@ -1,142 +1,141 @@
...
+or to configure your deployment tool so that it will bootstrap your monitor(s).
```
Comment (anthonyeleven) [inline doc/rados/configuration/auth-config-ref.rst:30]: Remove the parens.  Only the most degenerate of sandboxes only have one.

File: doc/rados/configuration/auth-config-ref.rst [hunk 4]
```diff
@@ -1,142 +1,141 @@
...
 
 Manual Deployment
 -----------------
 
-When you deploy a cluster manually, you have to bootstrap the monitor manually
-and create the ``client.admin`` user and keyring. To bootstrap monitors, follow
-the steps in `Monitor Bootstrapping`_. The steps for monitor bootstrapping are
-the logical steps you must perform when using third party deployment tools like
-Chef, Puppet,  Juju, etc.
+When you deploy a cluster manually, it is necessary to bootstrap the monitor
```
Comment (anthonyeleven) [inline doc/rados/configuration/auth-config-ref.rst:35]: monitors

File: doc/rados/configuration/auth-config-ref.rst [hunk 5]
```diff
@@ -146,70 +145,230 @@ Enablement
...
-The most common way to provide these keys to the ``ceph`` administrative
-commands and clients is to include a Ceph keyring under the ``/etc/ceph``
-directory. For Octopus and later releases using ``cephadm``, the filename
-is usually ``ceph.client.admin.keyring`` (or ``$cluster.client.admin.keyring``).
-If you include the keyring under the ``/etc/ceph`` directory, you don't need to
-specify a ``keyring`` entry in your Ceph configuration file.
+The most common way to make these keys available to ``ceph`` administrative
+commands and Ceph clients is to include a Ceph keyring under the ``/etc/ceph``
+directory. For Octopus and later releases that use ``cephadm``, the filename is
+usually ``ceph.client.admin.keyring`` or ``$cluster.client.admin.keyring``.  If
```
Comment (anthonyeleven) [inline doc/rados/configuration/auth-config-ref.rst:196]: Suggest removing the "or $cluster..."
We've deprecated custom cluster names for years at this point and it's reasonable to stop accounting for them.

**Review by anthonyeleven — APPROVED** [review]
The usual

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50949 - Reef backport
https://github.com/ceph/ceph/pull/50950 - Quincy backport
https://github.com/ceph/ceph/pull/50951 - Pacific backport

**Pull request merged**